### DISCUSSION TITLE
kampvuursessie

### DISCUSSION THREAD

**Discussion by Yu-Jing-van-Gerwen** [KaanKalmi/your-tribe-for-life-squad-page#45] [General]
dit zijn de vragen om te beantwoorden in de kampvuursessie zet onderin in de comments jouw eigen antwoorden

**Vragen die je met je team bespreekt:**

• Wat willen jullie opleveren?

• Hoe ver zijn jullie?

• Wat moet er nog gedaan worden?

• Hoe gaat de samenwerking?
• Wat gaat goed?
• wat kan beter?

**Comment by Yu-Jing-van-Gerwen** [comment]
**• Wat willen jullie opleveren?**
we willen een squadpage opleveren waarin je kan kiezen uit 2 pakketjes klas C en D en klik je op 1 van die pakketjes dan zie je de namen van de studenten in die klas

**• Hoe ver zijn jullie?**
de squadpage is bijna af er is 1 pull request open

**• Wat moet er nog gedaan worden?**
een readme moet geschreven worden en 1 pull request daar moet nog naar gekeken worden want de branch moet gemergd worden met de main

**• Hoe gaat de samenwerking?**
Goed we werken in branches  en we gebruiken het projectboard om ons project te managen ook bespraken we het werk op schooltijden 

**• Wat gaat goed?**
De samenwerking 

**• wat kan beter?**
de communicatie denk ik waarin we gericht te werk gaan en bespreken wie welke taak doet en wanneer we de voortgang van de taken bespreken om vervolgens het projectboard bij te werken waardoor we geen miscommunicatie hebben

**Comment by KaanKalmi** [comment]
- **Wat willen jullie opleveren?**
   - _We hebben een squad page opgeleverd met een leuke squad selectie animatie en hebben als concept gekozen voor pakketjes en brieven._
- **Hoe ver zijn jullie?**
    - _We zijn min of meer klaar met heel het project, we missen alleen een aantal should haves en could haves, maar de core functionaliteit is er wel._
- **Wat moet er nog gedaan worden?**
    - _Een aantal design fixes en animaties._
- **Hoe gaat de samenwerking?**
    - _Tijdens schooltijden: Prima, alles ging gewoon zoals het hoorde. Buiten schooltijden: Wat lastiger vooral ook omdat we een nieuw framework leren waardoor iedereen wel min of meer zat te leren hoe het nou precies werkt ipv coderen._
- **Wat gaat goed?**
    - _De project management, samenwerking op school i.v.m. bug fixes en elkaar helpen de code te begrijpen en verbeteren._
- **Wat kan beter?**
    - _De communicatie buiten schooltijden en de takenverdeling kon wel wat beter, maar dit komt ook doordat we een nieuw techniek zijn gaan leren in de tussentijd waardoor niemand echt snel te werk kon gaan en eerder gewoon continu de documentatie zat te lezen om te kijken hoe je nou iets moest doen._

**Comment by Lmikkers** [comment]
• Wat willen jullie opleveren?
Een werkende squadpage, je kunt een squad kiezen en daarna op een persoon klikken en naar die pagina gaan.

• Hoe ver zijn jullie?
We hebben een werkende squadpage gemaakt, we hebben de musts gemaakt, als we meer tijd hadden wilde we nog wat animaties toe voegen

• Wat moet er nog gedaan worden?
Een refactor van de code

• Hoe gaat de samenwerking?
De samenwerken verloopt goed, op school zoeken we elkaar op en bespreken we wat ieder heeft gedaan.  Het is soms alleen nog lastig, omdat we met een nieuwe technologie werken.
• Wat gaat goed?
De samenwerking vooral op school en projectmanagement.
• Wat kan beter?
Misschien de communicatie via teams en takenverdeling

**Comment by Robin1224** [comment]
• Wat willen jullie opleveren?
We hebben als team een werkende squad pagina kunnen opleveren, waarbij we de core van ons design hebben kunnen realiseren.

• Hoe ver zijn jullie?
Het project is grotendeels af. De must-haves zitten er in. We hebben niet alles wat in het originele design zat kunnen verwerken in ons eindproduct maar we hebben wel een werkende pagina kunnen leveren waarin de grote lijnen van het design aanwezig zijn.

• Wat moet er nog gedaan worden?
Er zijn nog een heleboel could-haves en should-haves waar we als groep niet aan toegekomen zijn, die eigenlijk wel in het originele design zaten.

• Hoe gaat de samenwerking?
• Wat gaat goed?
Over het algemeen verloopt de samenwerking fijn. Iedereen communiceert duidelijk, vooral op school, en heeft zijn taken afgekregen.

• wat kan beter?
Ik denk dat er meer gezamenlijk gewerkt kon worden. Uiteindelijk was het vaak zo dat iedereen een taak op zich nam, en deze gewoon thuis ging maken. Ik denk dat er regelmatig tegen dingen aangelopen werden waar we als groep snel uitgekomen zouden kunnen zijn. Ook denk ik dat er bepaalde momenten waren waarbij we misschien langer gezamenlijk hebben kunnen zitten om meer structuur te creëren in het werkproces, en dat bepaalde van deze meetings een beetje gerushed waren. Ten slotte denk ik dat we als team uitgebreider onze issues hebben kunnen documenteren, maar dit is iets waar wij ook vanuit de docenten al feedback op gekregen hebben.
